Sample/Program.cs

```diff
@@ -13,11 +14,41 @@ static async Task Main(string[] args)
             
             await client.GetGroupsAsync();
             await client.GetChatsAsync();
+            ICollection<Contact> contacts = await client.GetContactsAsync();
+            int x = contacts.Count;
+            while (x > 0)
+            {
+                var sortedContacts = client.Contacts().OrderByDescending(c => c.CreatedAtIso8601Time);
+                var earliestContact = sortedContacts.First();
+                ICollection<Contact> _contacts = await client.GetContactsAsync(true, earliestContact.CreatedAtIso8601Time);
```

Comment:
We definitely have clunkier APIs for paginated retrieval (Message comes to mind), but the big difference is that I think getting the list of all contacts is a reasonable request, unlike wanting all Messages at once, which is pretty edge-case. Can we add a GetAllContacts() method directly in the ApiClient that includes this logic built-in to simplify client handling? In the case that they want all contacts, that would just be a single line.